### PR TITLE
Reactor code in datasource-plugin

### PR DESCRIPTION
**Pull request by ZhangShenao**
## What is the purpose of the change

Reactor code in datasource-plugin

## Brief changelog

1. Extract common method to append where clause in `AbstractMapper`, in order to reduce duplicated code.
2. Remove unnecessary boxing in `ConfigInfoAggrMapperByMySql`.
3. Fix word spelling in `DiskUtils`.
4. Complete AbstractMapper Test Case.

### CODE REVIEW

File: plugin/datasource/src/main/java/com/alibaba/nacos/plugin/datasource/mapper/AbstractMapper.java [hunk 1]
```diff
@@ -42,28 +44,22 @@ public String select(List<String> columns, List<String> where) {
...
-            if (i != where.size() - 1) {
-                sql.append(" AND ");
-            }
-        }
+
+        appendWhereClause(where, sql);
         return sql.toString();
     }
-    
+
```
Comment (KomachiSion) [inline plugin/datasource/src/main/java/com/alibaba/nacos/plugin/datasource/mapper/AbstractMapper.java:55]: Please don't change the indent.

Use nacos code style to reformat codes.

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: plugin/datasource/src/main/java/com/alibaba/nacos/plugin/datasource/mapper/AbstractMapper.java (hunk 1 above)
Comment (ZhangShenao) [reply plugin/datasource/src/main/java/com/alibaba/nacos/plugin/datasource/mapper/AbstractMapper.java]: I've fixed check style.

File: plugin/datasource/src/test/java/com/alibaba/nacos/plugin/datasource/mapper/AbstractMapperTest.java [hunk 2]
```diff
@@ -35,7 +35,7 @@ public void setUp() throws Exception {
     @Test
     public void testSelect() {
         String sql = abstractMapper.select(Arrays.asList("id", "name"), Arrays.asList("id"));
-        Assert.assertEquals(sql, "SELECT id,name FROM tenant_info WHERE id = ?");
+        Assert.assertEquals(sql, "SELECT id,name FROM tenant_info  WHERE id = ?");
```
Comment (KomachiSion) [inline plugin/datasource/src/test/java/com/alibaba/nacos/plugin/datasource/mapper/AbstractMapperTest.java:38]: Why double blank?

File: plugin/datasource/src/test/java/com/alibaba/nacos/plugin/datasource/mapper/AbstractMapperTest.java [hunk 3]
```diff
@@ -59,12 +59,24 @@ public void testDelete() {
     @Test
     public void testCount() {
         String sql = abstractMapper.count(Arrays.asList("id"));
-        Assert.assertEquals(sql, "SELECT COUNT(*) FROM tenant_info WHERE id = ?");
+        Assert.assertEquals(sql, "SELECT COUNT(*) FROM tenant_info  WHERE id = ?");
```
Comment (KomachiSion) [inline plugin/datasource/src/test/java/com/alibaba/nacos/plugin/datasource/mapper/AbstractMapperTest.java:62]: Why double blank?

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: plugin/datasource/src/test/java/com/alibaba/nacos/plugin/datasource/mapper/AbstractMapperTest.java (hunk 2 above)
Comment (ZhangShenao) [reply plugin/datasource/src/test/java/com/alibaba/nacos/plugin/datasource/mapper/AbstractMapperTest.java]: fixed

File: plugin/datasource/src/test/java/com/alibaba/nacos/plugin/datasource/mapper/AbstractMapperTest.java (hunk 3 above)
Comment (ZhangShenao) [reply plugin/datasource/src/test/java/com/alibaba/nacos/plugin/datasource/mapper/AbstractMapperTest.java]: fixed

**Review by KomachiSion — APPROVED** [review]

**Comment by ZhangShenao** [comment]
@ready to close@

**Pull request merged**